ext/POSIX/t/sysconf.t

```diff
@@ -1,4 +1,4 @@
-#!perl -T
+#!perl -Tw
```

Comment:
Any particular reason for using `-w` instead of `use warnings`?

Reply:
> Any particular reason for using `-w` instead of `use warnings`?

Nothing other than the fact that we already have one switch (`-T`) on the shebang line of that file, so running with warnings only required one extra keystroke.

Reply:
`-w` does not run with warnings, it imposes warnings on anything that does not express a preference. It should not be used in general.